### DISCUSSION TITLE
Be Transparent About Who Gets Copilot for Free

### DISCUSSION THREAD

**Discussion by davecom** [community/community#28436] [Copilot Conversations]
As of August 5th, the [Copilot FAQ](https://github.com/pricing#i-work-on-open-source-projects-can-i-get-access-to-github-copilot-for-free) says:

> People who maintain popular open source projects receive a credit to have 12 months of GitHub Copilot access for free. A maintainer of a popular open source project is defined as someone who has write or admin access to one or more of the most popular open source projects on GitHub. Simply visit the GitHub Copilot subscription page to see if you are one of the open source maintainers that **meet our criteria** for a complimentary subscription.

Well, tell us what it means to "meet your criteria." Maintain at least one 1,000+ star project? Handle at least 5 pull requests a month?

**Comment by leecalcote** [comment]
Inquiring minds want to know.
